src/lio/bin/lio_mv.c

```diff
@@ -254,6 +261,7 @@ int main(int argc, char **argv)
             log_printf(15, "55555555555555555\n");
```

Comment:
Definitely!  That's one of probably 500 that need to get purged.  I 
started to remove some of them but then thought better of it.  I'm 
comfortable getting rid of a bunch of them in the old code but I've 
already found several issues in the converted code so I decided to leave 
them in for a while longer until we get a solid comfort level with the 
new base.